src/app/routes/onDemandAudio/getInitialData/index.test.ts

```diff
@@ -273,4 +290,22 @@ describe('Get initial data for on demand radio', () => {
     expect(countMissingFieldCalls(loggerMock.warn)).toBe(2);
     expect(countMissingFieldCalls(loggerMock.error)).toBe(2);
   });
+
+  it('should return media blocks in preparation for adding the media loader component', async () => {
+    fetchMock.mockResponse(JSON.stringify(onDemandRadioJson));
+    // @ts-expect-error partial data required for testing purposes
+    const { pageData } = await getInitialData({
+      path: 'mock-live-radio-path',
+      pageType: MEDIA_PAGE,
```

Comment:
Is this what we call on demand audio pages in this part of the code? Sounds a bit like media asset page. But also it might not matter here.